installer/functions/plugin/get-plugins.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { convertToBase64, listPluginsEndpoint } from './helper-plugins.private';
+import axios from 'axios';
+import { getParam, ACCOUNT_SID, AUTH_TOKEN } from '../common';
+
+/**
+ * This function is used to get all current plugins.
+ * If a plugin is not deployed via Docker, it will not be listed.
+ * Returns a list of plugins that have been deployed on the current account 
+ * provided we give it the SID and Auth Token
+ *  */ 
+export const handler: ServerlessFunctionSignature = async function(
+  context: Context,
+  event: {},
+  callback: ServerlessCallback
+) {
+
+  const response = new Twilio.Response();
+  response.appendHeader('Access-Control-Allow-Origin', '*');
+  response.setStatusCode(200);
+
+  try {
+    const accountSid = await getParam(context, ACCOUNT_SID);
+    const authToken = await getParam(context, AUTH_TOKEN);
+
+    console.log(accountSid, authToken);
```

Comment:
are `console.log`s the best way to log for serverless functions? is there any way we can utilize some logger that we can toggle on/off for production?

Reply:
also unsure if logging `authToken` is a good practice but i don't know enough about serverless